libs/wingc/src/capture.rs

```diff
@@ -328,7 +338,27 @@ fn scan_captures_in_expression(
 			res.extend(scan_captures_in_expression(lexp, env, statement_idx, diagnostics));
 			res.extend(scan_captures_in_expression(rexp, env, statement_idx, diagnostics));
 		}
-		ExprKind::Literal(_) => {}
+		ExprKind::Literal(lit) => match lit {
+			Literal::String(_) => {}
+			Literal::InterpolatedString(interpolated_str) => {
```

Comment:
Did you add interpolated string support here which was missing? This doesn't seem relevant directly to `var`.

Reply:
Correct, it was a missing case that I discovered while testing (bug). I'll mention it in the description